### DISCUSSION TITLE
Alexa integration

### DISCUSSION THREAD

**Discussion by phirestalker** [TandoorRecipes/recipes#2119] [General]
Has anyone heard of or started work on an Alexa integration for this?

**Comment by vabene1111** [comment]
not that i have heard of, but should be pretty easy as there are APIs for everytthing

**Comment by vednolacni** [comment]
Is there any update on this?

**Reply by smilerz** [reply]
Very low priority.

**Comment by Solear** [comment]
Alexa or another speech recignicion software would be very awesome.
I can't get my girlfriend to switch from bring to tandoor if she can't say “alexa, put milk on the shopping list” while standing at the fridge.
